os/board/rtl8721csm/src/component/soc/realtek/amebad/fwlib/ram_hp/rtl8721dhp_app_start.c

```diff
@@ -44,13 +44,15 @@ void app_init_psram(void);
 #ifdef CONFIG_PLATFORM_TIZENRT_OS
 extern void vPortEnterCritical(void);
 extern void vPortExitCritical(void);
-extern unsigned int _ebss;
+extern unsigned int _sidle_stack;
+extern unsigned int _sint_heap_start;
+extern unsigned int _sext_heap_start;
 extern unsigned int __StackLimit;
 extern unsigned int __PsramStackLimit;
-#define IDLE_STACK ((uintptr_t)&_ebss+CONFIG_IDLETHREAD_STACKSIZE-4)
-#define HEAP_BASE  ((uintptr_t)&_ebss+CONFIG_IDLETHREAD_STACKSIZE)
+#define IDLE_STACK ((uintptr_t)&_sidle_stack + CONFIG_IDLETHREAD_STACKSIZE - 4)
+#define HEAP_BASE  ((uintptr_t)&_sint_heap_start)
 #define HEAP_LIMIT ((uintptr_t)&__StackLimit)
-#define PSRAM_HEAP_BASE ((uintptr_t)&__psram_bss_end__[0])
+#define PSRAM_HEAP_BASE ((uintptr_t)&_sext_heap_start)
```

Comment:
Please update the value of `g_idle_topstack` in `os/arch/arm/src/amebad/amebad_start.c`

```
#define MSP_RAM_HP_NS           0x10004FFC
const uintptr_t g_idle_topstack = MSP_RAM_HP_NS;
```

Reply:
Updated the g_idle_topstack values. Thank you